inventory/hosts.example

```diff
@@ -12,8 +12,8 @@ ose3-master[1:3].test.example.com
 
 [nodes]
 ose3-master[1:3].test.example.com
-ose3-infra[1:2].test.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
-ose3-node[1:2].test.example.com openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
+ose3-infra[1:2].test.example.com openshift_node_labels={'region': 'infra', 'zone': 'default'}
```

Comment:
Variables provided on host line still require double-quote wrapping to delimit start/end.